notebooks/haystack-qa.ipynb

```diff
@@ -0,0 +1,1462 @@
+{
```

Comment:
<p>In essence, this notebook compares free and OS model   <code>bart_lfqa</code> and paid Open AI model <code>text-davinci-003</code>	for the long form generative QA task. </p><p>For the extractive qa task it tries <code>roberta-base-squad2</code>	. For all of these experiments, the retriever is BM25. </p><p> </p><p>Also, it has a separate experiment with combined <code>DPR</code> as retriever  and <code>facebook/rag-token-nq</code>	as the generator model.  </p><p> </p><p>Could we add this classification in the summary/conclusion? Once we have a validation dataset and metrics defined, we can add results of these experiments based on the metrics as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I think the detail level is appropriate now. </p><p> </p><blockquote>While it does have a free model for LFQA and an OpenAI version, the main goal of the notebook is to explore Haystack as a framework</blockquote><p> </p><p>That makes sense.</p>